client/src/hooks/useUser.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { UserContext } from '../context/user';
+
+export default async function loadUser(url) {
+  const [login, setLogin] = React.useState(false);
+  const [user, setUser] = React.useContext(UserContext);
+
+  const auth = localStorage.getItem('token');
+  try {
+    const res = await fetch(url, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Auth-Token': auth
+      }
+    });
+    const me = await res.json();
```

Comment:
isnt user better then me?

Reply:
when i use "user" it accesses the state, which i have to avoid here. had user in the beginning and then renamed it to me